assets/js/components/Sectors.vue

```diff
@@ -647,6 +647,17 @@ export default Vue.extend({
         .on('mouseover', this.tip.show)
         .on('mouseout', this.tip.hide);
 
+        aentered.append('use')
```

Comment:
Doing it in html would mean that the positioning of the icons would be absolute and the scaling would include a lot of media queries and hacky stuff